lib/matplotlib/artist.py

```diff
@@ -1267,8 +1267,7 @@ def aliased_name(self, s):
 
         if s in self.aliasd:
             return s + ''.join([' or %s' % x
-                                for x
-                                in six.iterkeys(self.aliasd[s])])
+                                for x in sorted(self.aliasd[s])])
```

Comment:
I think these are only called building the docs and as a failure mode in `getp`.